hack/.golint_failures

```diff
@@ -250,6 +250,7 @@ pkg/master
 pkg/master/controller/crdregistration
 pkg/master/tunneler
 pkg/printers
+pkg/printers/externalversion
```

Comment:
An externalversion package for printers should go under pkg/kubectl

Reply:
Yes, put all that under `pkg/kubectl/describe/`

Reply:
While doing so drop the `versioned` sub-directory from there and just put everything in `pkg/kubectl/describe`.

Reply:
I disagree. Moving this code into `pkg/kubectl` will increase the current internal, core dependencies. I believe we should continue working on `pkg/printers/externalversion` until it is free of internal dependencies--then move it into `pkg/kubectl`.